### PR TITLE
Imod6 991 add drn riv2 after refactor

### PR DESCRIPTION
**Pull request by HendrikKok**

### CODE REVIEW

**Comment by luitjansl** [comment]
this is a new version of pull request Imod6 991 add drn riv2; did you check the review comments I had on that?

File: imod_coupler/drivers/dfm_metamod/mf6_wrapper.py [hunk 1]
```diff
@@ -122,16 +122,55 @@ def get_river_stages(
         stage = bound[:, 0]
         return stage
 
-    def get_river_flux(
+    def set_correction_flux(
```
Comment (luitjansl) [inline imod_coupler/drivers/dfm_metamod/mf6_wrapper.py:125]: from a modflow perspective, this function sets the pumping rate of the wells in a well package. So I think the function should be called  "set_pumping_rate" , not " set_correction_flux" d

File: imod_coupler/drivers/dfm_metamod/mf6_wrapper.py [hunk 2]
```diff
@@ -122,16 +122,55 @@ def get_river_stages(
...
         self,
         mf6_flowmodel_key: str,
         mf6_package_key: str,
```
Comment (luitjansl) [inline imod_coupler/drivers/dfm_metamod/mf6_wrapper.py:128]: can you add a test for this function in the test_mf6_wrapper  file?

File: imod_coupler/drivers/dfm_metamod/mf6_wrapper.py (hunk 1 above)
Comment (HendrikKok) [reply imod_coupler/drivers/dfm_metamod/mf6_wrapper.py:125]: indeed

File: imod_coupler/drivers/dfm_metamod/mf6_wrapper.py (hunk 2 above)
Comment (HendrikKok) [reply imod_coupler/drivers/dfm_metamod/mf6_wrapper.py:128]: done

**Review by luitjansl — APPROVED** [review]

**Pull request merged**